docs/examples/userguide/fusedtypes/type_checking.py

```diff
@@ -0,0 +1,26 @@
+bunch_of_types = cython.fused_type(bytes, cython.int, cython.float)
```

Comment:
Unfortunatelly, you cannot add blank lines in the beginning. You must put white character to trick sphinx when you want to add blank lines in the beginning of file. But you cannot do that in .py files because code format check is complaining and yielding build failed. But in any case I did a commit to do my best to align both files...

Reply:
Fair enough. Doesn't seem like there's a good option here, so I'll try to ignore these